components/filter/filter.js

```diff
@@ -841,6 +847,15 @@ class Filter extends FocusMixin(LocalizeCoreElement(RtlMixin(LitElement))) {
 		));
 	}
 
+	async _handleExpandCollapse(e) {
+		const eventPromise = e.target.expanded ? e.detail.expandComplete : e.detail.collapseComplete;
+		const parentListItem = e.target.closest('d2l-list-item');
+		parentListItem.classList.add('expanding-content');
+
+		await eventPromise;
+		parentListItem.classList.remove('expanding-content');
+	}
```

Comment:
This seemed a bit weird so let me know if there's an alternative I'm missing. I tried `overflow-y: hidden` elsewhere and the list item was the only spot where it mattered.